cmd/kubeadm/app/cmd/cmd.go

```diff
@@ -93,5 +95,7 @@ func NewKubeadmCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 	experimentalCmd.AddCommand(phases.NewCmdPhase(out))
 	cmds.AddCommand(experimentalCmd)
 
+	utilfeature.DefaultFeatureGate.AddFlag(cmds.PersistentFlags())
```

Comment:
Yeah, I don't know much about `kubeadm`, but generally feature gates are reserved for server-side features, not client features.